pages/[lang]/index.js

```diff
@@ -0,0 +1,281 @@
+import Container from '../../components/layout/Container'
```

Comment:
I'm afraid about that approach with static props 😄 But now, please don't be scary. We can discuss that in next weeks, probably I will have to help you organize these pages.

Reply:
😄 Actually, next-i18next doesn't support i18n with next exports and SSG. I had to find a way around to make that work by pre-compiling the pages and serving them.